examples/federated_learning/fedavg/aggregation_worker/aggregate.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sedna.service.server import FedServer
+from torch import nn
+
+if __name__ == '__main__':
+    # run_server()
```

Comment:
Better to remove comments that are not used to explain the code

Reply:
updated